src/components/JobConfiguration.vue

```diff
@@ -328,7 +416,19 @@ ion-list {
 
   .mobile-only {
     display: none;
-  }  
+  }
+  .job-actions {
+    margin-top: var(--spacer-sm);
+    row-gap: var(--spacer-sm);
+    display: flex;
+    flex-wrap: wrap;
+  }
+  .job-actions > ion-item {
+    flex-basis: 50%;
+  }
+  .job-actions > ion-item > ion-checkbox{
+    margin: unset;
```

Comment:
Why do you need this? @azkyakhan

Reply:
Due to the margin on checkbox this item is looking like this :
![Screenshot from 2022-08-05 16-44-24](https://user-images.githubusercontent.com/82817616/183066813-13cb0b24-52aa-4d79-bf02-fc6822070a8c.png)

That is why we have unset the margin on checkbox.

Reply:
I don't see problem here with checkbox spacing. Am I missing something?

Reply:
That item line is slightly below from the adjacent item. 
![Untitled design](https://user-images.githubusercontent.com/82817616/183090225-ad91dc89-db11-49f1-a061-d66517f82870.png)

Reply:
Got it, what if we instead align content/items to end. I'd like to avoid changing item height from native behavior.

Reply:
I tried using `align-items` property but it is working when we are providing `flex-end` value to it :
![Screenshot from 2022-08-08 10-54-19](https://user-images.githubusercontent.com/82817616/183345399-b0e90dda-8fad-418b-b81e-264383003933.png)


If we are giving `end` as value then their is no change :
![Screenshot from 2022-08-08 10-54-46](https://user-images.githubusercontent.com/82817616/183345470-4be05af9-219f-44bf-98c1-0f710781ae03.png)

Reply:
Implemented align-items to end